Calendar/BookingHelpers/BookingMBHelpers.ps1

```diff
@@ -0,0 +1,238 @@
+﻿function Get-BookingMBData {
+    param (
+        [string]$Identity
+    )
+
+    $script:BMB = Get-Mailbox -Identity $Identity -ErrorAction SilentlyContinue
+    if ($null -eq $script:BMB) {
+        Write-DashLineBoxColor -Line "Booking Mailbox not found" -Color "Red"
+        return $null
+    }
+
+    $bookingMBData = [PSCustomObject]@{
+        DisplayName                   = $script:BMB.DisplayName
+        Identity                      = $script:BMB.Identity
+        RecipientType                 = $script:BMB.RecipientType  #: UserMailbox
+        RecipientTypeDetails          = $script:BMB.RecipientTypeDetails  # SchedulingMailbox
+        EmailAddresses                = $script:BMB.EmailAddresses
+        IsMailboxEnabled              = $script:BMB.IsMailboxEnabled
+        HiddenFromAddressListsEnabled = $script:BMB.HiddenFromAddressListsEnabled
+        IsSoftDeletedByRemove         = $script:BMB.IsSoftDeletedByRemove
+        IsSoftDeletedByDisable        = $script:BMB.IsSoftDeletedByDisable
+        IsInactiveMailbox             = $script:BMB.IsInactiveMailbox
+        WhenSoftDeleted               = $script:BMB.WhenSoftDeleted
+        WindowsEmailAddress           = $script:BMB.WindowsEmailAddress
+        WhenCreated                   = $script:BMB.WhenCreated
+        Guid                          = $script:BMB.Guid
+        OriginatingServer             = $script:BMB.OriginatingServer
+    }
+    return $bookingMBData
+}
+
+
+
+
+function Get-GraphBusiness {
+    param (
+        [string]$Identity
+    )
+
+    $MB = Get-MgBookingBusiness -BookingBusinessId $Identity
```

Comment:
>$MB [](http://example.com/codeflow?start=4&length=3)

$MB will likely make everyone think of Mailbox.
Please use a more descriptive name (unless this is the Booking MB).

Reply:
Here and else where.